pkg/api/controller_test.go

```diff
@@ -4122,6 +4123,352 @@ func TestStorageCommit(t *testing.T) {
 	})
 }
 
+func TestInterruptedImageManifest(t *testing.T) {
```

Comment:
Note that you need it to name it TestInjectInterruptedImageManifest for coverage due to -run ^TestInject here:

https://github.com/project-zot/zot/pull/381/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L57